src/components/faceting/faceting.tsx

```diff
@@ -634,27 +701,58 @@ const Faceting = ({
   return (
     <div className='side-panel-container' ref={sidePanelContainer}>
       <div className='faceting-columns-container'>
-        <Accordion className='panel-group' activeKey={activeKeys} alwaysOpen >
-          {reference.facetColumns.map((fc: any, index: number) => (
-            <Accordion.Item
-              eventKey={index + ''} key={index}
-              className={`facet-panel fc-${index}${facetModels[index].isOpen ? ' panel-open' : ''}`}
-            >
-              <Accordion.Header className={`fc-heading-${index}`} onClick={() => toggleFacet(index)}>
-                <FacetHeader
-                  displayname={fc.displayname}
-                  comment={fc.comment}
-                  isLoading={facetModels[index].isLoading}
-                  facetHasTimeoutError={facetModels[index].facetHasTimeoutError}
-                  noConstraints={facetModels[index].noConstraints}
-                />
-              </Accordion.Header>
-              <Accordion.Body>
-                {renderFacet(fc, index)}
-              </Accordion.Body>
-            </Accordion.Item>
-          ))}
-        </Accordion>
+        <DragDropContext onDragEnd={handleOnDragEnd}>
+          <ChaiseDroppable droppableId={'facet-droppable'}>
+            {
+              (provided: DroppableProvided) => (
```

Comment:
Is `DroppableProvided` documented somewhere in our code/docs? Probably isn't needed though since we can inspect it in `react-beautiful-dnd`

Reply:
I see there is a "similar" code block defined inside of this one for another `DraggableProvided`. Are both needed? If so, can you add a comment somewhere that explains why

Reply:
I changed the arguments to properly capture what they are (`draggableArgs` and `droppableArgs`). I also moved the fast list render to a different function so it's easier to follow the code.